src/xdocs/cmdline.xml.vm

```diff
@@ -89,9 +89,15 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           and column. You need to choose only one and recheck that it works. Usage of all of them is
           also ok, but might result in undesirable matching and suppress other issues.
         </li>
+        <li>
+          <code>-gxs,--generate-xpath-suppression</code> - generate to output a suppression.xml to
```

Comment:
this should be added to the first source in this section as it is a display of all the possible parameters users can give.